frontend/src/lib/components/ActivationSidebar/ActivationSidebar.tsx

```diff
@@ -0,0 +1,126 @@
+import { LemonButton, LemonButtonProps, LemonButtonWithSideAction } from '@posthog/lemon-ui'
+import clsx from 'clsx'
+import { useActions, useValues } from 'kea'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { activationLogic, Task } from './activationLogic'
+import './ActivationSidebar.scss'
+import { Progress } from 'antd'
+import { IconCheckmark, IconClose } from '../icons'
+import { SessionRecording as SessionRecordingConfig } from 'scenes/project/Settings/SessionRecording'
+import { ProfessorHog } from '../hedgehogs'
+
+const Task = ({ id, name, description, completed, canSkip, skipped, url }: Task): JSX.Element => {
```

Comment:
Rename `Task` to `TaskProps` or something different so that we don't redefine Task again